af/fastpath/fastpath/sshfeeder.py

```diff
@@ -78,7 +79,7 @@ def __init__(self, conf, hostname):
         self.sftp = ssh.open_sftp()
         # assert self.sftp.get_channel().get_transport()._preferred_compression[1] == 'zlib'
         self.new_downloads = []
-        self._archive_dir = f"/data/{self.hostname}/archive"
+        self._archive_dir = ARCHIVE_DIR
```

Comment:
Does this imply reading only from the oonified collectors?

Reply:
Yes